action.yml

```diff
@@ -53,20 +53,47 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: Check if review is requested
+      shell: python
+      env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
+        PR_STATE: ${{ github.event.pull_request.state }}
```

Comment:
🔍 Ensure the regex pattern covers all necessary cases for skipping reviews. Review the conditions to avoid false negatives.